rootfs/etc/services.d/logwatcher/run

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/with-contenv bash
```

Comment:
I don't think we need this file - looking at the adsbhub script, all logging will be to stdout/stderr.